packages/react-native-renderer/src/server/ReactNativeServerFormatConfig.js

```diff
@@ -231,6 +238,11 @@ export function writeEndCompletedSuspenseBoundary(
 ): boolean {
   return writeChunk(destination, END);
 }
+export function pushEndCompletedSuspenseBoundary(
```

Comment:
This one is named differently than the others but I think this is a better name. Because we don't know yet if all formats will have the same signature in the format. That's why `writeEnd(Pending/Completed/ClientRendered)SuspenseBoundary` is split even though it does the same thing. It also helps ensure that they get inlined.

Reply:
Oops unintentional but I'll update the DOM format config to also use this name.